packages/desktop/components/popups/CreateDelegationPopup.svelte

```diff
@@ -38,14 +38,15 @@
         $selectedWallet?.isTransferring
     $: validAmount = Big(rawAmount ?? 0)?.gt(0)
 
-    $: accountAddress, validAmount, prepareDelegationOutput()
+    $: accountAddress, validAmount, rawAmount, void prepareDelegationOutput()
```

Comment:
this is tricky, if you use the slider, how many times does `rawAmount`change and how many times are you actually then preparing the output? might be a bit overkilled